utils/ssh.go

```diff
@@ -91,3 +91,31 @@ func ParsePrivateKey(pemText, passphrase string) (ssh.Signer, error) {
 	}
 	return nil, err
 }
+
+// NoDeadlineConn is a net.Conn that ignores deadlines
```

Comment:
won't this make more sense ?

```suggestion
// NoDeadlineConn wraps a net.Conn to suppress "deadline not supported" errors from the crypto/ssh package.
```

Reply:
replaced